cidre/src/cv/buffer.rs

```diff
@@ -13,6 +13,12 @@ define_cf_type!(
     Buf(cf::Type)
 );
 
+impl Drop for Buf {
+    fn drop(&mut self) {
+        unsafe { CVBufferRelease(self) }
```

Comment:
Not sure we need `CVBufferRelease` function.
It is same as `CFRelease` but `null` safe.